src/redesign/_style.scss

```diff
@@ -632,24 +632,40 @@ select.form-control {
 }
 
 .alert .alert-icon {
```

Comment:
This shouldn't apply to all alert icons and instead, only apply to the email alert icon. For example, check the error icon in the following screenshot which is not aligned with the header text of the alert due to this:
<img width="516" alt="Screenshot 2021-06-29 at 2 31 06 PM" src="https://user-images.githubusercontent.com/2851134/123774379-c985d100-d8e6-11eb-9b2a-19ac7fa872e0.png">